docs/2.develop/contracts/whatiscontract.md

```diff
@@ -50,14 +56,9 @@ We envision that in the future, more languages will be supported and the support
 
 ---
 
-## Contract Primitives
-Contract primitives are the fundamental building blocks that can be combined to create a fully functional application. Blockchain primitives include Fungible Tokens (FT), Non Fungible Tokens (NFT), Decentralized Autonomous organizations (DAO), Link Drops and more.
-
-- [Fungible Tokens (FT)](/primitives/ft): Fungible token is representation of an asset on a blockchain that is interchangeable. Besides the native NEAR token, users can issue their own fungible tokens or use those that are already present in the ecosystem and created by other users or projects.
-
-- [Non Fungible Tokens (NFT)](/primitives/nft): In contrast with fungible tokens, non-fungible tokens (NFT) are unitary and therefore unique. This makes NFTs ideal to represent ownership of assets such as a piece of digital content, or a ticket for an event.
-
-- [Decentralized Autonomous organizations (DAO)](/primitives/dao): Decentralized Autonomous Organizations (DAOs) are self-organized groups that form around common purposes. Membership, decision making, and funding are coordinated by publicly voting on proposals through a smart contract.
-
-- [LinkDrops](/primitives/linkdrop): LinkDrops are an easy way to distribute digital assets (NFTs, FTs) via links. You provide a link for users and they can claim your drop.
+## Contract primitives
+Contract primitives such as FT, NFT and DAOs are fundamental building blocks that can be combined to create awesome user experiences such as reward tokens, decision-making tools and marketplaces. 
 
+:::tip
+Check our section on [primitives](/primitives/whatareprimitives) to learn more about them
```

Comment:
Question, should we start linking internal articles as URLs, or should we link as before, to relative markdown files path? e.g., `../contracts/whatiscontract.md`

Reply:
sometimes (and I don't know why yet) docusaurus complains when we use relative paths

Reply:
fixed